### DISCUSSION TITLE
Local Plugin?

### DISCUSSION THREAD

**Discussion by LandonSchropp** [parcel-bundler/parcel#5198] [General] [answered]
I really like parcel 2's new plugin interface, and I'd like to write a couple of custom transforms for my project. Since these transforms don't need to be shared with other projects, I'd prefer to keep them as local files in my repo.

Is there a way I can local plugins into Parcel from local files and continue to use it from the CLI?

**Comment by mischnic** [comment, marked as answer]
See https://github.com/parcel-bundler/parcel/issues/4321

Current "workaround": make your project a monorepo and create packages for the plugins

**Reply by LandonSchropp** [reply]
Thanks! I'll keep an eye on that issue.
